akka-actor/src/main/mima-filters/2.6.12.backwards.excludes/29985-actorRef-tell.excludes

```diff
@@ -0,0 +1 @@
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.actor.ActorRef.!")
```

Comment:
`ActorRef` requires to be extended by `InternalActorRef` which is an internal API, therefor I'd assume this should not be an issue.

Reply:
@patriknw I've compiled our project against 2.6.12 and configured tests with a local publish of this branch. All tests were ✅

Reply:
and you run the tests without recompiling the project (that is not trivial with sbt)?

Reply:
It is a sbt multimodule with the integration tests in a separate subproject, only for which I've overwritten the Akka dependencies. I've also verified the dependencyList on that project to be my build.

Reply:
thanks